datajoint/autopopulate.py

```diff
@@ -288,6 +288,9 @@ def _populate1(
                         exception=error.__class__.__name__,
                         msg=": " + str(error) if str(error) else "",
                     )
+                    logger.debug(
+                        f"Error making {key} -> {self.target.table_name} - {error_message}"
```

Comment:
I think `table_name` does not include the schema name whereas `full_table_name` does. 
```suggestion
                        f"Error making {key} -> {self.target.full_table_name} - {error_message}"
```